src/main/java/binnie/core/genetics/BreedingSystem.java

```diff
@@ -145,12 +144,15 @@ public void calculateArrays() {
 				allBranches.add(branch);
 			}
 		}
+
 		if (getSpeciesRoot().getMutations(false) != null) {
 			Set<IMutation> mutations = new LinkedHashSet<>();
 			mutations.addAll(getSpeciesRoot().getMutations(false));
 			if (this == Binnie.Genetics.beeBreedingSystem) {
-				mutations.addAll(ExtraBeeMutation.mutations);
+				// TODO !!! check for needed !!!
```

Comment:
Left just in case. I checked, all logics works fine.
Sooth to say, I don't know, why Binnie added this.